export/wiki/root.py

```diff
@@ -12,6 +12,7 @@
 from .stage_missions import MissionsStage
 from .stage_spawn_groups import SpawnGroupStage
 from .stage_species import SpeciesStage
+# from .stage_structures import StructuresStage
```

Comment:
Not intended to be included?

Reply:
Structure stage's intended to be included at later date to avoid more data changing commits in case changes are necessary.